packages/core/src/plugins/algolia.js

```diff
@@ -24,10 +24,10 @@ function addNoIndexClasses(content) {
     'dropdown',
     'b-modal',
     'panel:not([expanded])',
-    'popover div[slot=content]',
+    'span[data-mb-component-type="popover"] div[data-mb-slot-name="content"]',
     'question div[slot=hint]',
     'question div[slot=answer]',
-    'tab:not(:first-child)',
+    'tab:nth-of-type(n+2)',
```

Comment:
I'm assuming that we want to target all the instances of `tab` except for the first instance. As slot processing will introduce a new `template` node (which will become the first child instead of `tab`), this selector will not be correct. 

I think what we should be selecting is the second instance of `tab` onwards, and this selector lets us achieve that.

Reply:
let's move this (and above) to the other pr if necessary since we're just fixing the temp fouc styles here,
but both seem like separate pre-existing issues to me

For the second, it may be intended behaviour (e.g. in here https://markbind-master.netlify.app/userguide/usingcomponents#tabs - only the first (visible) one under `<tabs>` should be indexed)
Could look up the algolia pr, or git annotate this change to double check

Reply:
Alright! I've made the changes :-)